src/chapters/introduction-and-setup/setup.rst

```diff
@@ -36,7 +35,11 @@ The tools together make what is known as a **software development kit**, or **SD
 Unlike it's predecessor, just called .NET or .NET Framework, .NET Core is open-source and available on several 
 operating systems. The SDK provides the runtime environment and the virtual machine for compiling 
 and running C# programs. 
-.NET Core also notably contains the class library. This is the built-in code that takes care of common programming items,
+
+Starting with .NET 5, in November of 2020, the terms Core and Framework will disappear as there becomes just one
+version of .NET that is based on .NET Core.
```

Comment:
I'd suggest emphasizing the terms *Core* and *Framework*.

Also, can we clean up the phrasing around "...as there becomes..."?  Breaking this sentence up into smaller ones will help clarify the text.

Reply:
How about this?

Starting with .NET 5, in November of 2020, the terms _Core and Framework_ have been deprecated. There is now just one version of .NET that is based on .NET Core.

Reply:
The new phrasing is much better.  Let's run with that.